packages/elements-core/src/components/Docs/HttpOperation/Badges.tsx

```diff
@@ -34,6 +36,17 @@ export const DeprecatedBadge: React.FC = () => (
   </Tooltip>
 );
 
+export const InternalBadge: React.FC = () => (
+  <Tooltip
+    position={Position.BOTTOM_LEFT}
+    content="This endpoint/model is marked as internal and won't be visible in public docs."
```

Comment:
Maybe we should say operation instead of endpoint. The endpoint is /foo which might have a GET /foo operation and a deprecated POST /foo operation. We mix up these two things a lot in our ecosystem so it might be consistent with another mistaken use, but we deprecate operations not endpoints. If you spot that mistake anywhere else please fix it! 🍻